docs/content/en/docs/migrate/evolution-klt/_index.md

```diff
@@ -0,0 +1,192 @@
+where each development team runs,configures,
+and builds their own Git repository
+then uses tools such as Argo and Flux to push
+their microservice out to the target environment.
+This makes it easier for developers to work on their individual services:
+they become more innovative and deploy faster.
+but more complex for people who must operate these apps.
+
+Operations is now responsible for determining
+exactly what the application is --
+is it these five services in a certain version?
+Is it another flavor of the application
+with a different version of one service?
+Who is resposible for the individual tests
+as you start to deploy these different services together?
+
+GitOps deploys a service into the environment
+but testing, security checks, and validation are not part of GitOps.
+Different teams using different tools
+and different flavors of Kubernetes
+make it more complex for operations
+to validate the health of an application,
+ensuring that everything is secure, everything is orchestratable,
+and everything is observable.
+
+Keptn v1 can define metrics, response time, failure rates,
+and use a variety of tools
+to validate whether your overall deployment is actually healthy.
+But it has flaws in the GitOps world:
+
+- Single-pod ready does not mean application ready.
+  If you deploy different services in two different pods,
+  and they are ready,
+  it does not mean that your overall deployment is actually healthy
+
+- Very difficult to integrate Keptn into all the different tools,
+  especially with many development teams using different tools
+  because they had to trigger Keptn from each pipeline.
+  We saw that a serial integration approach would work better.
+
+- Keptn v1 does not use native Kubernetes CRDs
+  to configure SLIs/SLOs, tasks, and other thing
+  that are part of the environment where the application deployment lives.
+  Instead, it uses its own Git repo and its own configuration files.
+
+- As OpenTelementry and Prometheus have become more prevalent,
+  it makes sense to extract the data
+  and provide the results in OpenTelemetry and Prometheus
+  rather than storing the data in a special Keptn database
+  that can be accessed using Keptn APIs.
+
+- Keptn V1 can not define an application
+  as a sum of separate microservices,
+  each of which may be deployed at different times
+  using different tools.
+  This means that it cannot validate the full application.
+
+KLT solves these problems by providing an operator
+that can observe and orchestrate application-aware workload life cycles.
+This operator leverages Kubernetes webhooks
+and extends the Kubernetes scheduler
+to support pre- and post-deployment hooks.
+This means that when any deployment tool
+deploys a workload change into Kubernetes,
+KLT uses Kubernetes capabilities
+to do pre- and post-deployment checks.
+This means that:
+
+- Information is observable and is made available
+  using OpenTelemetry
+- We can provide traces that capture information
```

Comment:
"can" or "does"? If we "can", this should probably link to somewhere telling me how I do it.

Reply:
Added to source